drafts/sub-expressions-more-ex.Rmd

```diff
@@ -0,0 +1,73 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .Rmd
+      format_name: rmarkdown
+      format_version: '1.0'
+      jupytext_version: 0.8.3
+  kernelspec:
+    display_name: Python 3
+    language: python
+    name: python3
+  language_info:
+    codemirror_mode:
+      name: ipython
+      version: 3
+    file_extension: .py
+    mimetype: text/x-python
+    name: python
+    nbconvert_exporter: python
+    pygments_lexer: ipython3
+    version: 3.7.0
+---
+
+For each of the following expressions, write the sub-expressions beneath them. Remember, a sub-expression is a part of the code that is itself an expression.
+
+```{python}
+3 + 4 * 5
+```
+
+```{python}
+...
+```
+
+```{python}
+a + b * c
```

Comment:
Thats a good question... I would think that `a + b` would not be a sub-expression, because you couldn't reduce to it without disrupting the brackets. In other words, it would never be calculated (or its value used) when calculating the expression, so it does not make sense for it to be a sub-expression in my opinion.

Reply:
I agree with you agreeing!  But how to put that best in the explanation?